### ISSUE TITLE
Closing when clicking on scroll bar

### ISSUE THREAD

**Issue by fernandodamaso** [octopuscreative/FancySelect#65]
When the select has a max-height property and it creates a scrollbar, when clicking on the scrollbar to see the bottom items, the list disappear. Can the list only close when selecting one of the items only?

**Comment by BielRibeiro** [comment]
Same error here.

**Comment by victormiguez** [comment]
+1 ):

**Comment by DrLongGhost** [comment]
Seeing this in IE (9+, didn't test 8 or lower), Chrome & Safari. Only FF seems fine.

**Comment by devonbeard** [comment]
I just looked into this a little further and it only happens when the dropdown is below the trigger element. When it is above the element it works perfectly.

**Comment by davidpike7** [comment]
Having this issue in FF and IE too.

**Comment by devBwebi** [comment]
Same here! I hope it'll be fixed.
Couldn't achieve a solution.

**Comment by pepa-linha** [comment]
Yep, there is a problem with closing a options list when original select (tag <select>) lost focus, then options are hided. Fancy select is bad designed with that because tag with class "trigger" should be focusable but it is not, then tag with class with "options" should be child "trigger" element...

Fast solution is remove handling "blur.fs", in code

```javascript
sel.on('blur.fs', function() {

})
```

**Comment by devBwebi** [comment]
Thanks pepa-linha!
Have you tried it? is it working? I will give it a shot right away.
